proto/rill/admin/v1/api.proto

```diff
@@ -1677,4 +1677,5 @@ message AlertOptions {
   repeated string slack_webhooks = 11;
   // base64 state url of the dashboard when it was created
   string web_open_state = 12;
+  google.protobuf.Struct data_props = 13;
```

Comment:
1. Split into `resolver` and `resolver_properties` (to be serialized into `data:` in the YAML as `data[resolver] = resolverProperties`)
2. Move up above `query_name` and put `// Deprecated: ` after `query_name` and `query_args_json`